app/views/posts/show.html.erb

```diff
@@ -1,2 +1,12 @@
-<h1>Posts#show</h1>
-<p>Here is a single Post for a given user you are in '/users/user_id/posts'</p>
\ No newline at end of file
+<% content_for :title, "#{@user.name}'s Posts"%>
+<%= render "posts/user_post", post: @post %>
+<div class= "post-comments mt-4 border-2 border-gray-400">
+  <% @post.comments.each do |comment| %>
+    <div class="post-comment">
+      <p>
+        <span><strong><%= comment.author.name%></strong></span>
+        <span><%= comment.text %></span>
+      </p>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly add left and right margin to the post so that there is some space to match [the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAACBAEJCZCVZCZV6WBECSV2IOZBNG45A)

  ![image](https://user-images.githubusercontent.com/36542986/230017336-f080f2a7-93d0-4d2a-81b6-ac96a19434b1.png)